idemp-base-test/testing/request.xml

```diff
@@ -0,0 +1,30 @@
+<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:_0="http://idempiere.org/ADInterface/1_0">
+    <soapenv:Header/>
+    <soapenv:Body>
+        <_0:runProcess>
+            <_0:ModelRunProcessRequest>
+                <_0:ModelRunProcess>
+                    <_0:serviceType>RunTests_GW</_0:serviceType>
+                    <_0:ParamValues>
+                        <_0:field column="ChuBoe_Process_Loop_Count">
+                            <_0:val>1</_0:val>
+                        </_0:field>
+                        <_0:field column="ChuBoe_Process_Class_Filter">
+                            <_0:val>org.bandahealth.idempiere.base.test</_0:val>
+                        </_0:field>
+                    </_0:ParamValues>
+                </_0:ModelRunProcess>
+                <_0:ADLoginRequest>
+                    <_0:user>SuperUser</_0:user>
```

Comment:
Just updating this here from our conversation earlier today: while we could have it in the `.env` file, it proves more of a hassle since the process is directly associated with a hard-coded client in a migration script (and must be so). So, we'll leave this as-is.